src/Common/tests/System/Net/Http/Http2LoopbackConnection.cs

```diff
@@ -578,7 +587,7 @@ public async Task SendDefaultResponseAsync(int streamId)
             await WriteFrameAsync(headersFrame).ConfigureAwait(false);
         }
 
-        public async Task SendResponseHeadersAsync(int streamId, bool endStream = true, HttpStatusCode statusCode = HttpStatusCode.OK, bool isTrailingHeader = false, IList<HttpHeaderData> headers = null)
+        public async Task SendResponseHeadersAsync(int streamId, bool endStream = true, HttpStatusCode statusCode = HttpStatusCode.OK, bool isTrailingHeader = false, bool endHeaders = true, IList<HttpHeaderData> headers = null)
```

Comment:
would it make sense to instead take flags directly? i.e. `FrameFlags flags = FrameFlags.EndHeaders`